src/component.js

```diff
@@ -1,12 +1,10 @@
-import { componentNew } from 'js-component-tools';
+import { componentEmbed, componentNew } from '../../js-component-tools/dist/api.mjs';
```

Comment:
This file does not exist inside this codebase - should this be reverted to the package name?
```suggestion
import { componentEmbed, componentNew } from 'js-component-tools';
```